Frontend/src/components/MentorDashboard.jsx

```diff
@@ -0,0 +1,12 @@
+import { useState} from 'react'
+import './MentorProfile.css'
+
+function MentorDashboard() {
+    return(
+        <div>
+            <h1>Mentor Dashboard</h1>
+        </div>
+    )
+}
+
+export default MentorDashboard
```

Comment:
might be able to just combine the import into `export default function MentorDashboard() {...}` ?

Reply:
Resolved